Description: Fix an unaligned memory access in the dba extension
Origin: http://svn.php.net/viewvc?view=revision&revision=295764
Last-Update: 2010-05-01

Index: php/ext/dba/dba.c
===================================================================
--- php.orig/ext/dba/dba.c
+++ php/ext/dba/dba.c
@@ -250,7 +250,7 @@ static size_t php_dba_make_key(zval *key
 	zval *key;													\
 	char *key_str, *key_free;									\
 	size_t key_len; 											\
-	int skip = 0;  												\
+	long skip = 0;  											\
 	switch(ac) {												\
 	case 2: 													\
 		if (zend_parse_parameters(ac TSRMLS_CC, "zr", &key, &id) == FAILURE) { \
